### ISSUE TITLE
Fakenet Issue

### ISSUE THREAD

**Issue by geeksailor** [REMnux/remnux-cli#154]
Devs,
Was testing out Fakenet today and was hitting some errors. I noticed that the it attempts to run it using Python 2.7. However when I checked the git site for the project, they just ported the project to Python3. Looking at the salt package, it appears to install Fakenet and it's dependencies using python2. I think this might be causing some issues with the program. Thought it might be something worth looking into.

Also, Mandiant [https://github.com/mandiant/flare-fakenet-ng](url) appears to have take over the git for this project from Fireeye. The link for the salt package still goes to the old git  but I think it auto redirects to the Mandiant anyways so it may be moot.  But just in case you feel like changing the link.

Cheers!

**Comment by digitalsleuth** [comment]
Good catch. I'll take a look at that this evening and submit the updates (after testing). 

Cheers!

**Comment by digitalsleuth** [comment]
Hi @geeksailor , I went over the current Python 3 fakenet version, and it's still in Alpha right now. I ran a few tests with installation and practice, and there are a few issues which are present and still unfixed by the repo owner. 

I'm going to take another approach over the next few days to see if Mandiant will fix the issues, and I'll see if I can come up with some form of a workaround until they do.

Cheers

**Comment by geeksailor** [comment]
@digitalsleuth Yeah the whole alpha thing confused me considering what is installed through Remnux says is Fakenet 3.0 according to pip.Not sure if that is meant to be v3.0a.

All good though. Do what you do and I'm sure you will figure it out one way or another. Just wanted to make sure the issue was caught and brought up to ensure Remnux is running it's best self.

Please feel free to close this ticket out once you think everything is good and the project is happy with the solution.
Cheers!

**Comment by digitalsleuth** [comment]
A recent update in fakenet (py3 version) may resolve this issue. Will look into it further.

**Comment by digitalsleuth** [comment]
Hi @geeksailor , I know it's been a long time coming but Fakenet has been updated and a PR submitted to merge it back into REMnux. Cheers!
